src/prerender.ts

```diff
@@ -248,7 +248,7 @@ export async function prerender(nitro: Nitro) {
     interval: nitro.options.prerender.interval,
   });
 
-  await nitro.hooks.callHook("prerenderer:finished", {
+  await nitro.hooks.callHook("prerenderer:close", {
```

Comment:
`:done` hook would be better. By convention in nuxt and unjs we use `close` for disposal step when closing resources.